plugins/trace_api_plugin/trace_api_plugin.cpp

```diff
@@ -39,6 +46,36 @@ namespace {
          _log.log(fc::log_message( context, detail_string ));
       }
    }
+
+   template<typename F>
+   void emit_killer(F&& f) {
+      try {
+         f();
+      } catch (const yield_exception& ) {
```

Comment:
Add comment that `yield_exception` is thrown for any exception of extraction store.